src/site/GlobalNavigation.css

```diff
@@ -0,0 +1,243 @@
+/* GLOBAL NAVIGATION */
+
+nav#dg_slide-menu a {
+  text-decoration: none;
+}
+
+.i-float-right {
```

Comment:
think this is a little off. when you name a class like this, it suggest it will be floated right, but in this case it will also change the color and font. You have two options, make this selector more specific to what is is `.awesome-button` or break this up.

Reply:
Updated. Will push up with next commit. Let me know if this naming convention fits the element better, or if you would like to see it revised.